Page50135.APIV1PriceListHeader.al

```diff
@@ -0,0 +1,50 @@
+page 50135 "Price List Header Entity"
+{
+    PageType = API;
+    Caption = 'Price List Headers';
+    APIPublisher = 'Opmetrix';
+    APIGroup = 'opmetrix';
+    APIVersion = 'v1.0';
+    EntityName = 'priceListHeader';
+    EntitySetName = 'priceListHeaders';
+    DelayedInsert = true;
+
+    SourceTable = "Price List Header";
+    //SourceTable = 7000;
+    ODataKeyFields = Code;
+
+    layout
+    {
+        area(Content)
+        {
+            field("Code"; "Code") { ApplicationArea = All; }
+            field("Description"; "Description") { ApplicationArea = All; }
+            field("SourceGroup"; "Source Group") { ApplicationArea = All; }
+            field("SourceType"; "Source Type") { ApplicationArea = All; }
+            field("SourceNo"; "Source No.") { ApplicationArea = All; }
+            field("ParentSourceNo"; "Parent Source No.") { ApplicationArea = All; }
+            field("SourceID"; "Source ID") { ApplicationArea = All; }
+            field("PriceType"; "Price Type") { ApplicationArea = All; }
+            field("AmountType"; "Amount Type") { ApplicationArea = All; }
+            field("CurrencyCode"; "Currency Code") { ApplicationArea = All; }
+            field("StartingDate"; "Starting Date") { ApplicationArea = All; }
+            field("EndingDate"; "Ending Date") { ApplicationArea = All; }
+            field("PriceIncludesVAT"; "Price Includes VAT") { ApplicationArea = All; }
+            //field("VAT Bus. Posting Gr. (Price)"; "VAT Bus. Posting Gr. (Price)") { ApplicationArea = All; }
```

Comment:
Why are there some fields commented out?  Just not needed, or do they have issues?